dns/octodns-docker/requirements.txt

```diff
@@ -1,25 +1,42 @@
+atomicwrites==1.3.0
+attrs==19.3.0
 cachetools==3.1.1
-certifi==2019.6.16
+certifi==2019.9.11
 chardet==3.0.4
+coverage==4.5.4
 dnspython==1.16.0
-futures==3.2.0
-google-api-core==1.13.0
-google-auth==1.6.3
-google-cloud-core==1.0.2
-google-cloud-dns==0.30.1
+google-api-core==1.14.3
+google-auth==1.7.1
+google-cloud-core==1.0.3
+google-cloud-dns==0.31.0
 googleapis-common-protos==1.6.0
 idna==2.8
+importlib-metadata==0.23
 incf.countryutils==1.0
-ipaddress==1.0.22
-natsort==6.0.0
-octodns==0.9.5
-protobuf==3.8.0
-pyasn1==0.4.5
-pyasn1-modules==0.2.5
-python-dateutil==2.6.1
-pytz==2019.1
-PyYAML==5.1.1
+ipaddress==1.0.23
+more-itertools==7.2.0
+natsort==6.2.0
+octodns==0.9.9
+packaging==19.2
+pluggy==0.13.0
+pprintpp==0.4.0
+protobuf==3.10.0
+py==1.8.0
+pyasn1==0.4.7
+pyasn1-modules==0.2.7
+pycountry==19.8.18
+pycountry-convert==0.7.2
+pyparsing==2.4.5
+pytest==5.2.2
```

Comment:
Looks like there's some unneeded test dependencies.

Try starting up a new python:3-alpine container, running a `pip install octodns==0.9.9`, and then a `pip freeze` and copy the output into this file

Reply:
Ready, I added the google-cloud-dns dependecy as well.